### PR TITLE
Update troubleshooting page for Visual Studio Tools for Unity

### PR DESCRIPTION
**Pull request by sailro**
cc @jbevain

### CODE REVIEW

File: docs/cross-platform/troubleshooting-and-known-issues-visual-studio-tools-for-unity.md [hunk 1]
```diff
@@ -46,14 +46,25 @@ For FMOD, there is a workaround, you can pass FMOD_STUDIO_INIT_SYNCHRONOUS_UPDAT
...
 
+### Extra reloads, or Visual Studio losing all open windows
+Be sure to never touch project files directly from an asset processor or any other tool. If you really need to participate to the project file content, we expose an API for that. Please check the [Assembly references issues section](#Assembly-reference-issues).
+
+If you experience extra reloads or if Visual Studio is losing all open Windows on reload, please make sure that you have proper .NET targeting packs installed. Please check the following section about frameworks for more information.
+
+
+### On Windows, Visual Studio asks to download the Unity target framework
+Visual Studio Tools for Unity requires the .net framework 3.5, which isn't installed by default on Windows 8 or 10. To fix this issue, follow the instructions to download and install the .net framework 3.5.
```
Comment (gewarren) [inline docs/cross-platform/troubleshooting-and-known-issues-visual-studio-tools-for-unity.md]: .NET Framework (note capitalizaiton, in two places)

File: docs/cross-platform/troubleshooting-and-known-issues-visual-studio-tools-for-unity.md [hunk 2]
```diff
@@ -46,14 +46,25 @@ For FMOD, there is a workaround, you can pass FMOD_STUDIO_INIT_SYNCHRONOUS_UPDAT
 ### Incompatible project in Visual Studio
 First, check that Visual Studio is set as your external script editor in Unity (Edit/Preferences/External Tools). Then check that the Visual Studio   plugin is installed in Unity (Help/About must display a message like Microsoft Visual Studio Tools for Unity is enabled at the bottom). Then check that the extension is properly installed in Visual Studio (Help/About).
 
+### Extra reloads, or Visual Studio losing all open windows
+Be sure to never touch project files directly from an asset processor or any other tool. If you really need to participate to the project file content, we expose an API for that. Please check the [Assembly references issues section](#Assembly-reference-issues).
```
Comment (gewarren) [inline docs/cross-platform/troubleshooting-and-known-issues-visual-studio-tools-for-unity.md]: "If you really need to manipulate the project file, we expose an API for that." - does this sound better?

Comment (sailro) [reply docs/cross-platform/troubleshooting-and-known-issues-visual-studio-tools-for-unity.md]: Clearly. Thanks

File: docs/cross-platform/troubleshooting-and-known-issues-visual-studio-tools-for-unity.md (hunk 1 above)
Comment (sailro) [reply docs/cross-platform/troubleshooting-and-known-issues-visual-studio-tools-for-unity.md]: Done. Thanks

**Comment by sailro** [comment]
Ready to have this merged. Thanks

**Pull request merged**